packages/flutter/lib/fix_data.yaml

```diff
@@ -17,6 +17,294 @@
 
 version: 1
 transforms:
+  # Changes made in https://github.com/flutter/flutter/pull/97972/
+  - title: "Migrate 'ThemeData.toggleableActiveColor' to individual themes"
+    date: 2022-05-18
+    element:
+      uris: [ 'material.dart' ]
+      constructor: ''
+      inClass: 'ThemeData'
+    changes:
+    - kind: 'removeParameter'
+      name: 'toggleableActiveColor'
+    - kind: 'addParameter'
+      index: 95
```

Comment:
These indices don't seem to match. They should be the index of the parameter in the constructor.